### PR TITLE
Moving tests to kinto_changes/tests

### PR DESCRIPTION
**Pull request by vsham20**

### CODE REVIEW

**Comment by Natim** [comment]
It seems that you didn't used ``git mv`` so that ``tests/`` files where not removed.
Also you will need to change the way we run tests.

**Comment by vsham20** [comment]
I have made changes according to test cases, but i don't understand why Travis is still failing?

File: kinto_changes/tests/tests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
```
Comment (Natim) [inline kinto_changes/tests/tests/__init__.py]: Why is this file now in tests/tests/ ?

File: tox.ini [hunk 2]
```diff
@@ -2,7 +2,7 @@
 envlist = py27,py34,kinto-master,flake8
 
 [testenv]
-commands = py.test tests --cov-report term-missing --cov-fail-under 100 --cov kinto_changes {posargs}
+commands = py.test kinto_changes/tests --cov-report term-missing --cov-fail-under 100 --cov kinto_changes {posargs}
```
Comment (Natim) [inline tox.ini]: kinto_changes should be enough here.

File: tox.ini [hunk 3]
```diff
@@ -15,7 +15,7 @@ deps =
     webtest
 
 [testenv:kinto-master]
-commands = py.test tests --cov-report term-missing --cov-fail-under 100 --cov kinto_changes {posargs}
+commands = py.test kinto_changes/tests --cov-report term-missing --cov-fail-under 100 --cov kinto_changes {posargs}
```
Comment (Natim) [inline tox.ini]: kinto_changes should be enought here.

**Pull request closed without merging**